shell/common/language_util_linux.cc

```diff
@@ -4,14 +4,34 @@
 
 #include "shell/common/language_util.h"
 
-#include "ui/base/l10n/l10n_util.h"
+#include "base/i18n/rtl.h"
+
+#if defined(USE_GLIB)
```

Comment:
glib is always enabled for linux https://source.chromium.org/chromium/chromium/src/+/main:build/config/ui.gni;l=35-39, the checks in `ui/base` are needed since they are reused by chrome builds that don't support glib. I would suggest to remove this condition.